CHANGELOG.md

```diff
@@ -1,91 +1,96 @@
 # Change Log
 
+## [0.4.3] (October 10, 2023)
```

Comment:
Just based on previous history, not sure this is sufficient for a new release. `dj_config` wouldn't be included in the PyPI release, so it comes down to if we want pip users to have the update from #659 

```suggestion
## [0.4.3] (Unreleased)
```